frontend/javascripts/libs/math.js

```diff
@@ -0,0 +1,5 @@
+// @noflow
+// This module should be used to access the Math object, so it can be mocked in the unit tests
+// mockRequire("libs/math", myFakeMath);
+
+export default Math;
```

Comment:
I'd rename this file to `uid_generator.js` or something like that and then expose the request id generator, which can be mocked. This would limit the surface area of the mocked code a bit. Now, the mock for the save saga would need to be adapted as soon as we use more `Math` methods in the save saga (similar to how `Math.min` had to be re-added to the mock). Also, it suggests to always use this module instead of importing Math directly, but that's not really necessary in most cases.

Reply:
Yes, that's much better, thank you! :)